### PR TITLE
update placement enhancement

### PR DESCRIPTION
**Pull request by elgnay**
Signed-off-by: Yang Le <yangle@redhat.com>

### CODE REVIEW

File: enhancements/sig-architecture/6-placements/README.md [hunk 1]
```diff
@@ -416,26 +333,14 @@ Limitations:
...
-7. If `NumberOfClusters` is less than the size of the intersection, select random ManagedClusters from the intersection and return;
-8. Otherwise, return ManagedClusters in the intersection with some random additional ManagedClusters from the remaining of the new matched ManagedClusters;
-
-**Note:** The old matched ManagedClusters are preferred during the predicate evaluation, which makes the placement decisions as stable as possible.
+1. Get all clustersetbindings in the placement namespace;
+2. Get eligible clustersets for the placement. A clusterset is eligible for a placement if it is bound to the placement namespace, and it is the .spec.clustersets if it is not empty;
+3. Find out all available clusters for the placement;
+4. Filter clusters with cluster predicates;
+5. Select clusters and generate cluster decisions;
+6. Bind the cluster decisions into placementdecisions;
```
Comment (qiujian16) [inline enhancements/sig-architecture/6-placements/README.md:343]: step 6 is not clear. Saying  bind... is confusing. Maybe step 5 only is sufficient.

File: enhancements/sig-architecture/6-placements/metadata.yaml [hunk 2]
```diff
@@ -8,5 +8,5 @@ approvers:
   - "@pmorie"
   - "@mdelder"
 creation-date: 2021-01-28
-last-updated: 2021-03-19
+last-updated: 2021-06-28
 status: provisional
```
Comment (qiujian16) [inline enhancements/sig-architecture/6-placements/metadata.yaml:12]: eof

File: enhancements/sig-architecture/6-placements/README.md (hunk 1 above)
Comment (elgnay) [reply enhancements/sig-architecture/6-placements/README.md]: Step 6 removed.

File: enhancements/sig-architecture/6-placements/metadata.yaml (hunk 2 above)
Comment (elgnay) [reply enhancements/sig-architecture/6-placements/metadata.yaml]: Added.

File: enhancements/sig-architecture/6-placements/metadata.yaml [hunk 3]
```diff
@@ -8,5 +8,5 @@ approvers:
   - "@pmorie"
   - "@mdelder"
 creation-date: 2021-01-28
-last-updated: 2021-03-19
-status: provisional
\ No newline at end of file
+last-updated: 2021-06-28
+status: provisional
```
Comment (qiujian16) [inline enhancements/sig-architecture/6-placements/metadata.yaml:12]: should this be  changed to implemeted?

Comment (elgnay) [reply enhancements/sig-architecture/6-placements/metadata.yaml]: Updated.

**Pull request merged**